DTS-STN/canadian-dental-care-plan
# Remove Confirm page for mailing address flow

### Description

 - Remove Confirm page for mailing address flow

### Related Azure Boards Work Items
[AB#3127](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3127)

### Checklist


### Test Instructions
- Navigate to http://localhost:3000/en/personal-information/mailing-address/edit
- Upon clicking 'save' it should redirect to personal-information.
- No data is saved. (future implementation)
